spec/unit/actions/package_update_spec.rb

```diff
@@ -30,5 +30,63 @@ module VCAP::CloudController
         expect(package).to have_annotations({ key_name: 'tokyo', value: 'grapes' })
       end
     end
+
+    describe 'update docker credentials' do
+      let(:body) do
+        {
+          username: 'Udo',
+          password: 'Won\'t tell you'
+        }
+      end
+      let(:package) { PackageModel.make(type: 'docker', docker_image: 'image-magick.com') }
+      let(:message) { PackageUpdateMessage.new(body) }
+
+      it 'updates the package\'s docker credentials' do
```

Comment:
When using a `'` inside the text, it's better (more readable) to use `"` for the whole text instead of escaping:
```
it "updates the package's docker credentials" do
```